src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -157,6 +157,11 @@ public InferenceAnnotatedTypeFactory(
         }
     }
 
+    protected VariableAnnotator createVariableAnnotator(BaseAnnotatedTypeFactory realTypeFactory, InferrableChecker realChecker,
```

Comment:
Can we have a brief javadoc on this new method?

Reply:
@topnessman  Ping here:-) It would be nice to have a javadoc for this new method. Specifically, describe the purpose of this method, and how sub-classes are expected to override this method under which situation.